src/_scss/pages/state/overview/_overview.scss

```diff
@@ -10,6 +10,10 @@
             width: rem(64);
             height: auto;
             margin-right: rem(28);
+
+            & img {
+                border: 1px solid #CCCCCC;
```

Comment:
`img` doesn't need the ampersand in front here, since it is below `state-overview__flag` in the DOM tree.
Can you use the variable we have for `$color-gray-lighter`?